requirements/pip.txt

```diff
@@ -21,7 +21,7 @@ jsonfield==2.0.2
 requests==2.21.0
 requests-toolbelt==0.9.1
 slumber==0.7.1
-pyyaml==3.13
+pyyaml==5.1
```

Comment:
Looks like they skipped the 4 here :)

https://github.com/yaml/pyyaml/blob/e471e86bf6dabdad45a1438c20a4a5c033eb9034/CHANGES#L7-L34

Nothing breaking, just remove support for old versions of python and start supporting 3.7, those are the biggest changes.